packages/s2-core/src/utils/interaction/merge-cells.ts

```diff
@@ -244,18 +278,21 @@ const removeUnmergedCellsInfo = (
  * @param removedCells
  * @param sheet
  */
-export const unmergeCell = (removedCells: MergedCells, sheet: SpreadSheet) => {
-  if (removedCells) {
-    const newMergedCellsInfo = removeUnmergedCellsInfo(
-      removedCells,
-      sheet.options?.mergedCellsInfo,
-    );
-    if (newMergedCellsInfo?.length !== sheet.options?.mergedCellsInfo?.length) {
-      sheet.setOptions({
-        mergedCellsInfo: newMergedCellsInfo,
-      });
-      removedCells.remove(true);
-    }
+export const unmergeCell = (sheet: SpreadSheet, removedCells: MergedCell) => {
+  if (!removedCells || removedCells.cellType !== CellTypes.MERGED_CELLS) {
+    // eslint-disable-next-line no-console
+    console.error('unmergeCell: the cell is not a MergedCell');
```

Comment:
可以把 `removedCells`打印一下
```suggestion
    console.error(`unmergeCell: the cell ${removedCells.xxx} is not a MergedCell`);
```

Reply:
log信息都走 `DebuggerUtil`  类